src/reducers/index.js

```diff
@@ -205,17 +205,17 @@ function reducers(state = getInitialState(), action) {
  * @param {!Object} state
  * @private
  */
-function initializeRowHeights(state) {
+function initializeRowHeightsAndOffsets(state) {
```

Comment:
renamed this since it also updates the row offsets (which is present in the description though)